api/v1alpha1/openapi.yaml

```diff
@@ -1187,14 +1228,21 @@ components:
     Snapshot:
       type: object
       properties:
-        inventory:
-          $ref: '#/components/schemas/Inventory'
         createdAt:
           type: string
           format: date-time
+        status:
+          type: string
+          enum: [pending, processing, ready, failed]
+          description: Status of the snapshot processing
+        error:
+          type: string
+          description: Error message if snapshot processing failed
+        inventory:
+          $ref: '#/components/schemas/Inventory'
       required:
-        - inventory
```

Comment:
This breaks backward compatibility. But in this case it should be OK, if we merge UI PR together.